src/robots/robot-wrapper.cpp

```diff
@@ -44,6 +44,16 @@ namespace tsid
       m_Md.setZero(m_model.nv);
       m_M.setZero(m_model.nv, m_model.nv);
     }
+    RobotWrapper::RobotWrapper(const pinocchio::Model& m, bool verbose)
```

Comment:
The argument `verbose` is not used here.

Reply:
I did. Thanks for finding it.